packages/cli/README.md

```diff
@@ -11,3 +11,1048 @@ npm install @stacks/cli
+
+Example:
+
+    $ stx get_confirmations e41ce043ab64fd5a5fd382fba21acba8c1f46cbb1d7c08771ada858ce7d29eea
+    {
+      "blockHeight": 567890,
+      "confirmations": 7,
+    }
+
+
+```
+### get_zonefile
+```shell script 
+Get a zone file by hash.
+
+Example:
+
+    $ stx get_zonefile ee77ad484b7b229f09461e4c2b6d3bd3e152ba95
+    $ORIGIN ryanshea.id
+    $TTL 3600
+    _http._tcp URI 10 1 "https://gaia.blockstack.org/hub/15BcxePn59Y6mYD2fRLCLCaaHScefqW2No/1/profile.json"
+
+
+```
+### make_zonefile
+```shell script 
+Generate a zone file for a Blockstack ID with the given profile URL.  If you know the ID-address for the Blockstack ID, the profile URL usually takes the form of:
+
+     {GAIA_URL_PREFIX}/{ADDRESS}/profile.json
+
+where `{GAIA_URL_PREFIX}` is the *read* endpoint of your Gaia hub (e.g. https://gaia.blockstack.org/hub) and `{ADDRESS}` is the base58check part of your ID-address (i.e. the string following 'ID-').
+
+Example:
+
+     $ stx make_zonefile example.id ID-1ArdkA2oLaKnbNbLccBaFhEV4pYju8hJ82 https://my.gaia.hub/hub
+     $ORIGIN example.id
+     $TTL 3600
+     _http._tcp      IN      URI     10      1       "https://my.gaia.hub/hub/1ArdkA2oLaKnbNbLccBaFhEV4pYju8hJ82/profile.json"
+
+
+```
+### zonefile_push
+```shell script 
+Push a zone file on disk to the Blockstack peer network.  The zone file must correspond to a zone file hash that has already been announced.  That is, you use this command in conjunction with the `register`, `update`, `renew`, or `name_import` commands.
+
+Example:
+
+    $ stx -H https://core.blockstack.org zonefile_push /path/to/zonefile.txt
+    [
+      "https://core.blockstack.org"
+    ]
+
+
+```
+## Authentication
+### authenticator
+```shell script 
+Run an authentication endpoint for the set of names owned by the given backup phrase.  Send applications the given Gaia hub URL on sign-in, so the application will use it to read/write user data.
+
+You can supply your encrypted backup phrase instead of the raw backup phrase.  If so, then you will be prompted for your password before any authentication takes place.
+
+Example:
+
+    $ export BACKUP_PHRASE="oak indicate inside poet please share dinner monitor glow hire source perfect"
+    $ export APP_GAIA_HUB="https://1.2.3.4"
+    $ export PROFILE_GAIA_HUB="https://hub.blockstack.org"
+    $ stx authenticator "$APP_GAIA_HUB" "$BACKUP_PHRASE" "$PROFILE_GAIA_HUB" 8888
+    Press Ctrl+C to exit
+    Authentication server started on 8888
+
+```
+## Account Management
+### balance
+```shell script 
+Query the balance of an account.  Returns the balances of each kind of token that the account owns.  The balances will be in the *smallest possible units* of the token (i.e. satoshis for BTC, microStacks for Stacks, etc.).
+
+Example:
+
+    $ stx balance 16pm276FpJYpm7Dv3GEaRqTVvGPTdceoY4
+    {
+      "BTC": "123456"
+      "STACKS": "123456"
+    }
+    $ stx balance SPZY1V53Z4TVRHHW9Z7SFG8CZNRAG7BD8WJ6SXD0
```

Comment:
I suggest using 1) testnet addresses unless absolutely necessary to use mainnet and 2) real addresses so ideally if someone copy / pastes any one of these example commands, it mostly works.

Reply:
Added these changes.